contrib/READMEs/EMISSARY-COMMANDS.md

```diff
@@ -0,0 +1,215 @@
+# Emissary Commands
+
+### Help
+
+If you want to see more detailed information on a command, add the command name after help.  For example, see all the 
+arguments with descriptions for the *what* command, run:
+
+```
+./emissary help what
+```
+
+### Common parameters
+
+The rest of commands all have *(-b or --projectBase)* arguments that can be set, but it must match PROJECT_BASE.
+The config directory is defaulted to <projectBase>/config
+but can also be passed in with *(-c or --config)*.  When running from the git checkout, you should use
+*target* as the projectBase.  Feel free to modify config files in target/config before you start.
+
+See the *help -c <commandName>* for each command to get more info.
+
+### What
+
+This command will use the configured engines to identify the file.  Emissary currently only comes with the 
+SizeIdPlace, so the id will TINY or SMALL etc.  See that class for more info.  The *-i or --input* argument
+is required as well as *-b*.  Here is how to run the command
+
+```
+./emissary what -i <path to some file>
+```
+
+### Server (Standalone)
+
+This command will start up an Emissary server and initialize all the places, 
+a pickup place, and drop off filters that are configured.  It will start in 
+standalone mode if *-m or --mode* is not specified.  By default, the number of 
+MobileAgents is calculated based on the specs of the machine.  On modern computers, 
+this can be high.  You can control the number of agents with *-a or --agents*.  Here 
+is an example run.
+
+```
+./emissary server -a 2
+```
+
+Without further configuration, it will start on http://localhost:8001.  If you browse to that 
+url, you will need to enter the username and password defined in target/config/jetty-users.properties,
+which is emissary and emissary123.
```

Comment:
Should we remove the username and password? I understand it is a default, but this seems like bad practice.

Reply:
I think it is ok to have the username and password. If someone goes to start with all defaults then they will have to dig into the config to see what the default is.